security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -161,95 +161,4 @@ This is the general availability (GA) release of the {cert-manager-operator}.
-
-* You can enable monitoring and metrics collection for the {cert-manager-operator} by using a service monitor to perform the custom metrics scraping. After you have enabled monitoring for the {cert-manager-operator}, you can query its metrics by using the {product-title} web console. For more information, see xref:../../security/cert_manager_operator/cert-manager-monitoring.adoc#cert-manager-monitoring[Enabling monitoring for the {cert-manager-operator}]
-
-[id="cert-manager-operator-1.10.2-bug-fixes"]
-=== Bug fixes
-
-* Previously, the `unsupportedConfigOverrides` field replaced user-provided arguments instead of appending them. Now, the `unsupportedConfigOverrides` field properly appends user-provided arguments. (link:https://issues.redhat.com/browse/CM-23[*CM-23*])
-+
-[WARNING]
-====
-Using the `unsupportedConfigOverrides` section to modify the configuration of an Operator is unsupported and might block cluster upgrades.
-====
-
-* Previously, the {cert-manager-operator} was installed as a cluster Operator. With this release, the {cert-manager-operator} is now properly installed as an OLM Operator. (link:https://issues.redhat.com/browse/CM-35[*CM-35*])
-
-[id="cert-manager-operator-1.10.2-known-issues"]
-=== Known issues
-
-* Using `Route` objects is not fully supported. Currently, to use {cert-manager-operator} with `Routes`, users must create `Ingress` objects, which are translated to `Route` objects by the Ingress-to-Route Controller. (link:https://issues.redhat.com/browse/CM-16[*CM-16*])
-
-* The {cert-manager-operator} does not support using Azure Active Directory (Azure AD) pod identities to assign a managed identity to a pod. As a workaround, you can use a service principal to assign a managed identity. (link:https://issues.redhat.com/browse/OCPBUGS-8665[*OCPBUGS-8665*])
-
-* The {cert-manager-operator} does not support using Google workload identity federation. (link:https://issues.redhat.com/browse/OCPBUGS-9998[*OCPBUGS-9998*])
-
-* When uninstalling the {cert-manager-operator}, if you select the *Delete all operand instances for this operator* checkbox in the {product-title} web console, the Operator is not uninstalled properly. As a workaround, do not select this checkbox when uninstalling the {cert-manager-operator}. (link:https://issues.redhat.com/browse/OCPBUGS-9960[*OCPBUGS-9960*])
+* After uninstalling the {cert-manager-operator} by using the web console, the {cert-manager-operator} does not remove the cert-manager controller, CA injector, and Webhook pods automatically from the `cert-manager` namespace. As a workaround, you can manually delete the cert-manager controller, CA injector, and Webhook pod deployments present in the `cert-manager` namespace. (link:https://issues.redhat.com/browse/OCPBUGS-13679[*OCPBUGS-13679*])
```

Comment:
Should the Jira have the release notes field populated (such as Release Note Type = Known issue)?

Reply:
@swghosh did an engg bug fix this compatibility? If yes, we would need to incorporate the above comment by @dfitzmau

Reply:
@dfitzmau regarding your second comment above, it is an existing content that you would see in line 164. Strangely, github has detected changes. All this PR is does is removal of text.

Reply:
Yeah, typically the following release note fields on a Jira would be populated:

* Release Note Text: Can use note from this PR. 
* Release Note Type: Known issue
* Release Note Status: `In progress` is probably the best status. Set `Done` when the known issue is fixed.

Reply:
Ah, OK. Please ignore my second point. Shubha. I can see that the primary target content has been removed.